functions/webhooks/line/FlexToLine.protected.ts

```diff
@@ -27,32 +27,30 @@ export type Body = Partial<WebhookEvent> & {
   recipientId?: string; // The Line id of the user that started the conversation. Provided as query parameter
 };
 
-const sendLineMessage = (context: Context<EnvVars>) => async (
```

Comment:
@murilovmachado Can you confirm if this is the final version, or if this is outdated?

Reply:
@GPaoloni @acedeywin  Strange that this PR deletes this line. This function should exist.